tsconfig.json

```diff
@@ -71,6 +71,7 @@
     "tests/**/*",
     "tools/**/*",
     "integration/**/*",
-    ".ng-dev/**/*"
+    ".ng-dev/**/*",
+    "~/**/*", // Windows Circle CI yarn cache
```

Comment:
This shouldn't be needed.  The working directory is `~/ng` so `~/.cache` is outside the root of the checked out repository.

Reply:
In Windows we end up with the following structure

```

Mode                LastWriteTime         Length Name
----                -------------         ------ ----
d-----         4/5/2021  11:50 AM                .circleci
d-----         4/5/2021  11:51 AM                .git
d-----         4/5/2021  11:50 AM                .github
d-----         4/5/2021  11:51 AM                .husky
d-----         4/5/2021  11:50 AM                .idea
d-----         4/5/2021  11:50 AM                .ng-dev
d-----         4/5/2021  11:50 AM                .yarn
d-----         4/5/2021  11:50 AM                bin
d-----         4/5/2021  11:50 AM                docs
d-----         4/5/2021  11:50 AM                etc
d-----         4/5/2021  11:50 AM                integration
d-----         4/5/2021  11:50 AM                lib
d-----         4/5/2021  11:51 AM                node_modules
d-----         4/5/2021  11:50 AM                packages
d-----         4/5/2021  11:50 AM                scripts
d-----         4/5/2021  11:50 AM                tests
d-----         4/5/2021  11:50 AM                tools
d-----         4/5/2021  12:11 PM                ~
-a----         4/5/2021  11:50 AM             23 .bazelignore
-a----         4/5/2021  11:50 AM           5863 .bazelrc
-a----         4/5/2021  11:50 AM            211 .bazelversion
-a----         4/5/2021  11:50 AM            282 .editorconfig
-a----         4/5/2021  11:50 AM            228 .gitattributes
-a----         4/5/2021  11:50 AM            856 .gitignore
-a----         4/5/2021  11:50 AM           2536 .mailmap
-a----         4/5/2021  11:50 AM           4601 .monorepo.json
-a----         4/5/2021  11:50 AM              6 .nvmrc
-a----         4/5/2021  11:50 AM              9 .prettierignore
-a----         4/5/2021  11:50 AM             73 .prettierrc
-a----         4/5/2021  11:50 AM            160 .yarnrc
-a----         4/5/2021  11:50 AM            400 BUILD.bazel
-a----         4/5/2021  11:50 AM          13203 CONTRIBUTING.md
-a----         4/5/2021  11:50 AM             36 Dockerfile
-a----         4/5/2021  11:50 AM           1073 LICENSE
-a----         4/5/2021  11:50 AM           8567 package.json
-a----         4/5/2021  11:50 AM          12471 README.md
-a----         4/5/2021  11:50 AM           1471 renovate.json
-a----         4/5/2021  11:50 AM            414 tsconfig-test.json
-a----         4/5/2021  11:50 AM           2680 tsconfig.json
-a----         4/5/2021  11:50 AM           4264 tslint.json
-a----         4/5/2021  11:50 AM           3667 WORKSPACE
-a----         4/5/2021  11:50 AM         597933 yarn.lock
```

Which causes 

```
Error: Package @angular/dev-infra-private found in "C:\\Users\\circleci\\ng\\~\\.cache\\yarn\\v6\\.tmp\\cc3298f9f5ce96bfe9d2a4b1937a26b2", not found in .monorepo.json.
```

Still trying to figure out why.